### ISSUE TITLE
Icons in intellisence are oversize

### ISSUE THREAD

**Issue by hjkcai** [Microsoft/vscode#64957]
- VSCode Version: 1.30.0
- OS Version: macOS 10.13.4

![image](https://user-images.githubusercontent.com/731326/49920419-21a18700-fee5-11e8-8943-5676034d1362.png)

Sorry but I still wanna ask. Is it a bug or feature? As you can see, there is almost no whitespace left between the border and the candidate. Maybe a little bit of whitespace make it more comfortable.

**Comment by jrieken** [comment]
please share the suggest settings with us

**Comment by jrieken** [comment]
please share the suggest settings with us

**Comment by hjkcai** [comment]
Do you mean `settings.json`? There is no way to configure the icon size. The css is hardcoded.

**Comment by jrieken** [comment]
Yes - it seems that you customised font size, line height, and/or font (?) because it certainly doesn't look like that on my machine

**Comment by hjkcai** [comment]
Yes. I customized fonts:

```json
{
  "editor.fontFamily": "Inconsolata, Heiti SC",
  "editor.fontSize": 14,
  "editor.lineHeight": 23
}
```

The icons are smaller if I remove these settings. But they are still blurry because they are not exactly 16px. 

![image](https://user-images.githubusercontent.com/731326/49937308-4c093980-ff11-11e8-9136-60c1ccf81e44.png)

I suggest always set the icons to 16px because they are designed to be 16px.

**Comment by fwcd** [comment]
I have encountered the same problem, especially on non-retina monitors the icons appear less sharp than the surrounding code...

**Comment by hjkcai** [comment]
@jrieken Hi, how is this issue going? I hope this could be fixed in 1.30.1.

**Comment by fwcd** [comment]
![image](https://user-images.githubusercontent.com/30873659/50089158-4ce9e600-0205-11e9-8100-ddab65496add.png)

A comparison of the current icon size (on the left) with the previous one (on the right).
